### PR TITLE
Adds checks for S3Bucket Access Keys

### PR DESCRIPTION
**Pull request by neelp03**
### Resolves part of #922 

```js
const s3BucketKeys = require('config.json').S3Bucket;
const { ACCESS_KEY, SECRET_KEY} = s3BucketKeys;

if(ACCESS_KEY!= 'NOT_SET' && SECRET_KEY != 'NOT_SET') {
  executeFile();
} else {
   doNothing();
   //if its a promise then resolve
}
```
> Note: current default value: 'XXXX...X'    --->    New default value: 'NOT_SET' in config.example.json

### CODE REVIEW

File: api/cloud_api/util/S3BucketApiHandler.js [hunk 1]
```diff
@@ -8,9 +8,14 @@ class S3BucketApiHandler {
       secretAccessKey: s3BucketKeys.AWSSECRETKEY,
       region: 'us-west-1',
     });
+    /* eslint-disable-next-line */
+    this.hasValidKeys = (s3BucketKeys.AWSACCESSKEYID != 'NOT_SET' && s3BucketKeys.AWSSECRETKEY != 'NOT_SET') ? true : false;
```
Comment (evanugarte) [inline api/cloud_api/util/S3BucketApiHandler.js:12]: how about doing the if statement like the google api handler does?

if (both not set) 
  stuff

File: api/peripheral_api/routes/Printer.js [hunk 2]
```diff
@@ -23,7 +22,10 @@ router.get('/healthCheck', (req, res) => {
 
 const s3 = new AWS.S3({ apiVersion: '2012-11-05' });
 router.post('/sendPrintRequest', async (req, res) => {
-
+  /* eslint-disable-next-line */
+  if (s3BucketKeys.AWSACCESSKEYID == 'NOT_SET'&& s3BucketKeys.AWSSECRETKEY == 'NOT_SET') {
```
Comment (evanugarte) [inline api/peripheral_api/routes/Printer.js:26]: you can make this if statement multi lines to avoid the lint error
if (condition1
  && condition2) {

**Review by evanugarte — COMMENTED** [review]
merge dev into ur branch as well pls

File: api/peripheral_api/routes/Printer.js (hunk 2 above)
Comment (allen-tran) [inline api/peripheral_api/routes/Printer.js:26]: also add a space in front of && to keep the standard 💯

File: api/cloud_api/util/S3BucketApiHandler.js [hunk 3]
```diff
@@ -11,6 +11,10 @@ class S3BucketApiHandler {
   }
   getSignedUrl(s3Params, bucket) {
     return new Promise((resolve, reject) => {
+      if(s3BucketKeys.AWSACCESSKEYID == 'NOT_SET'
+      && s3BucketKeys.AWSSECRETKEY == 'NOT_SET') {
+        return resolve(true);
+      }
```
Comment (evanugarte) [inline api/cloud_api/util/S3BucketApiHandler.js:17]: @allen-tran we should use this class for writing data to s3 for printer

**Review by evanugarte — APPROVED** [review]

File: api/cloud_api/util/S3BucketApiHandler.js (hunk 3 above)
Comment (allen-tran) [reply api/cloud_api/util/S3BucketApiHandler.js:17]: ok

**Review by allen-tran — APPROVED** [review]
great work

**Pull request merged**